torchgeo/transforms/indices.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""TorchGeo indices transforms."""
+
+from typing import Dict
+
+import torch
+from torch import Tensor
+from torch.nn import Module  # type: ignore[attr-defined]
+
+# https://github.com/pytorch/pytorch/issues/60979
+# https://github.com/pytorch/pytorch/pull/61045
+Module.__module__ = "torch.nn"
+
+
+epsilon = 1e-10
```

Comment:
```suggestion
_EPSILON = 1e-10
```
Should be a private global variable